lib/galaxy/tools/actions/upload_common.py

```diff
@@ -396,8 +377,10 @@ def _chown(path):
             # TODO: This will have to change when we start bundling inputs.
             # Also, in_place above causes the file to be left behind since the
             # user cannot remove it unless the parent directory is writable.
-            if link_data_only == 'copy_files' and trans.app.config.external_chown_script:
-                _chown(uploaded_dataset.path)
+            if link_data_only == 'copy_files':
```

Comment:
```suggestion
            if link_data_only == 'copy_files' and trans.user:
```
should fix the selenium tests.